src/cgns_header.h

```diff
@@ -510,6 +527,38 @@ typedef struct {            /* FlowEquationSet_t Node               */
     cgns_model *emconduct;  /* ptrs to in-mem. copy of EMConductivityM. */
 } cgns_equations;
 
+typedef struct {            /* ParticleGoverningEquations_t node    */
+    char_33 name;           /* name of ADF node                     */
+    double id;              /* ADF ID number (address) of node      */
+    cgns_link *link;        /* link information         */  /* V2.1 */
+    int in_link;            /* set if child of a linked node        */
+    CGNS_ENUMT(ParticleGoverningEquationsType_t) type;
+    int ndescr;             /* no of Descriptor_t nodes             */
+    cgns_descr *descr;      /* ptrs to in-memory copy of descr      */
+    int nuser_data;         /* number of user defined data nodes    */  /* V2.1 */
```

Comment:
@MicK7 , Do you know if we need to indicate the version here? This is a new struct, and obviously, older versions of CGNS are not applicable. I'm assuming this was included to suggest what version the new field was added.

Reply:
I don't think we need the version here since this kind of node is already available but it would be nice to add a version for the top structure  to  tell that `ParticleGovervinvEquation_t`  was introduced in  `/* V4.5 */` 
I need to check but I think that something similar was done for the FamilyTree CPEX.